src/domain/discounts/details/discount-rule/index.js

```diff
@@ -249,6 +243,26 @@ const DiscountRuleModal = ({
               <Text fontWeight="500">Item</Text>
             </Pill>
           </Flex>
+          {isAllocatedToItem && (
```

Comment:
You should also me allowed to choose items even though the allocation method is total. This would just mean, that if you have two items in the cart where only one is applicable for the applied discount, then only this would be reflected in the total discount. This is support on the backend.